packages/web-components/src/global/_variables.scss

```diff
@@ -31,6 +31,7 @@ $color-gray-lightest: #f1f1f1;
 $color-gray-warm-dark: #494440;
 $color-gray-warm-light: #e4e2e0;
 $color-gray-cool-light: #dce4ef;
+$color-blue-cool-lightest: #d9e8f6;
```

Comment:
You should see at the top of this file that it says:

`// Do not edit directly`

This file should be generate from the css-library package and I think @powellkerry just added the blue color tokens to it from his PR. Kerry, can yo confirm?